pkg/provider/provider.go

```diff
@@ -43,10 +43,14 @@ type DeploymentResource struct {
 
 // NewDeploymentResource returns DeploymentResource with default values.
 func NewDeploymentResource() *DeploymentResource {
+	// Set NGINX_SERVICE_TYPE to LoadBalancer.
+	DefaultDeploymentVars := map[string]string{}
+	DefaultDeploymentVars["NGINX_SERVICE_TYPE"] = "LoadBalancer"
+
 	return &DeploymentResource{
 		DeploymentFiles:       []string{},
 		FlagDeploymentVars:    map[string]string{},
-		DefaultDeploymentVars: map[string]string{},
+		DefaultDeploymentVars: DefaultDeploymentVars,
```

Comment:
why not just do:
```suggestion
		DefaultDeploymentVars: map[string]string{
		  "NGINX_SERVICE_TYPE" : "LoadBalancer",
		}
```
?

Reply:
This is also possible. Thanks